PhysicsTools/NanoAOD/plugins/TriggerOutputBranches.cc

```diff
@@ -32,11 +36,17 @@ void TriggerOutputBranches::updateTriggerNames(TTree& tree,
   for (unsigned int j = 0; j < newNames.size(); j++) {
     std::string name = newNames[j];  // no const & as it will be modified below!
     std::size_t vfound = name.rfind("_v");
-    if (vfound != std::string::npos && (name.compare(0, 3, "HLT") == 0 || name.compare(0, 2, "L1") == 0)) {
+    if (vfound != std::string::npos &&
+        ((name.compare(0, 3, "HLT") == 0) || name.compare(0, 2, "L1") == 0 ||
+         std::find(m_triggerBranchesToKeep.begin(),
+                   m_triggerBranchesToKeep.end(),
+                   name.replace(vfound, name.size() - vfound, "")) != m_triggerBranchesToKeep.end())) {
       name.replace(vfound, name.size() - vfound, "");
     }
     bool found = false;
-    if (name.compare(0, 3, "HLT") == 0 || name.compare(0, 4, "Flag") == 0 || name.compare(0, 2, "L1") == 0) {
+    if ((name.compare(0, 3, "HLT") == 0) || name.compare(0, 4, "Flag") == 0 || name.compare(0, 2, "L1") == 0 ||
+        (std::find(m_triggerBranchesToKeep.begin(), m_triggerBranchesToKeep.end(), name) !=
+         m_triggerBranchesToKeep.end())) {
```

Comment:
how "save the branch if Scouting is in the name" ? to avoid having to add m_triggerBranchesToKeep and the corresponding customise command

Reply:
OK, sounds good!